cmd/lnmuxd/run.go

```diff
@@ -48,6 +48,27 @@ func runAction(c *cli.Context) error {
 		return err
 	}
 
+	group, ctx := errgroup.WithContext(context.Background())
```

Comment:
Another option would be to change this to something like:
```
backCtx, cancel := context.WithCancel(context.Background())
defer cancel()

group, ctx := errgroup.WithContext(backCtx)
```
and then if we return early, we cancel the error group.

Reply:
The problem remains the same no? 
Since we don't have a `errgroup.Wait()` we will not wait for all errgroup goroutines (so we will not wait the server to close gracefully)